provider-example/src/hpke.rs

```diff
@@ -54,7 +93,9 @@ impl HpkeRs {
 
 impl Debug for HpkeRs {
     fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
-        f.debug_struct("HpkeRsHpke").finish()
+        f.debug_struct("HpkeRs")
```

Comment:
Nit: maybe we can just derive this now?

Reply:
Switched to a derived `Debug` in the commit that reworked `HpkeRs` to only hold a `HpkeSuite`. That's the earliest point the change became possible.